src/util.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import kebabCase from 'lodash.kebabcase';
+
+export function getColorClass( colorSlug, colorContextName ) {
+	if ( ! colorSlug || ! colorContextName ) {
+		return undefined;
+	}
+
+	return `has-${ kebabCase(
+		colorSlug
+	) }-${ colorContextName } has-text-color`;
+}
+
+export function getColorCode( colorCode, colorProps, scope ) {
+	if ( colorProps !== undefined && colorProps.style !== undefined ) {
+		return colorProps.style[ scope ];
+	}
+
+	if ( colorCode !== undefined ) {
+		return colorCode;
```

Comment:
Personally prefer shorthand for these, but up to you:

```
return colorCode || 'currentColor';
```

Reply:
Your suggestion looks much better! I also prefer shorthand syntax. Suggestion applied. 🙌